.github/workflows/build-and-deploy-prod.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build a docker image, push it to ghcr.io, and deploy it to an Azure WebApp.
+name: Build and Deploy -- PROD
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  determine-trigger:
+    name: Determine if this was triggered by a release or workflow_dispatch
+    runs-on: ubuntu-latest
+    outputs:
+      is-release: ${{ env.IS_RELEASE }}
+    steps:
+      - name: Check if this was triggered by a release
+        id: release
+        run: |
+          echo "IS_RELEASE"=${{ github.event_name == 'release' }} >> $GITHUB_ENV
+
```

Comment:
This is no longer needed here.  The check was moved to the shared workflows in clearlydefined/operations.  See [clearlydefined/service/PR#1120](https://github.com/clearlydefined/service/pull/1120/files)
```suggestion
```